.travis.yml

```diff
@@ -28,23 +28,13 @@ install:
   - pip install -U https://github.com/netjson/django-netjsongraph/tarball/master
   - python setup.py -q develop
   - pip install -r requirements-test.txt
-
-before_script:
-  - flake8
-  - isort --check-only --recursive --diff .
+  # Temporary: remove when openwisp-utils 0.4 is released
+  - pip install -e git+git://github.com/openwisp/openwisp-utils#egg=openwisp_utils[qa]
```

Comment:
I just released openwisp-utils 0.3.1 which includes the latest changes on openwisp-qa-check, please try to remove this line to see if it works without it. This applies to all outstanding similar PRs.